charts/modules/apps/cert-manager/values.yaml

```diff
@@ -148,11 +243,69 @@ crossplane-aws-iam:
                 Condition:
                   ArnLike:
                     "aws:PrincipalArn": 'arn:aws:iam::{{ .Values.global.awsAccountId }}:role/{{ include "common-gitops.names.release" . }}'
+      xrole:
+        enabled: '{{ eq .Values.global.deploymentType "issuer" }}'
```

Comment:
This is correct and matches with what we finalized yesterday. This means, issuer deployment should be able to create xrole in the dnsZoneAwsAccount